src/Simulation/Simulators/QCTraceSimulator/QCTraceSimulator.Primitive.Measure.cs

```diff
@@ -5,10 +5,11 @@ namespace Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.Implementati
 {
     using System;
     using Microsoft.Quantum.Simulation.Core;
+    using Microsoft.Quantum.Intrinsic.Interfaces;
 
     public partial class QCTraceSimulatorImpl
     {
-        public virtual Result Measure__Body(IQArray<Pauli> paulis, IQArray<Qubit> targets)
```

Comment:
nit: Could we keep the access modifiers on the method definitions throughout this PR? I believe without specifying the access, it will default to `private`, but making that explicit goes a long way with readability.

Reply:
See my other answer, where it turns out that adding access modifiers to fully qualified signatures is not supported.